Catalog/Examples/UI/RxUIMenu/RxUIMenu.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  RxUIMenu.swift
+//  Catalog
+//
+//  Created by Sven Svetina on 14.03.2023..
+//  Copyright © 2023 Infinum. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+// MARK: - RxUIMenu -
+
+/// A container for grouping related menu elements in an app menu or contextual menu.
+///
+/// Create RxUIMenu wrapper around the UIMenu objects and use them to construct the menus and submenus your app displays.
+/// Used in case where reactive action is needed.
+@available(iOS 15.0, *)
+class RxUIMenu<T> {
+
+    let title: String
+    let image: UIImage?
+    let children: [RxUIAction<T>]
```

Comment:
Can children be a non-action? If not, I would go with `actions` instead.

Reply:
This is named after Apple's own naming on UIMenu so it kind of made sense to keep it this way :)